server/api/team/team.controller.js

```diff
@@ -73,6 +74,66 @@ exports.players = function(req, res) {
     });
 };
 
+exports.fill = function(req, res) {
+    var currentSport = ""
+    var user = req.user;
+      if (user.role !== admin) {
+        logger.log("error", "Filling teams is restricted to admins only");
+        return res.send (403);
+      }
+    Sport.find(req.params.id).then(function(sport) {
+        currentSport = sport
+    }).then(function() {
+        Team.findAll({
+            where: [{special: null, sportId: currentSport.id}]
```

Comment:
Since you mentioned the URL will be something along the lines of : ```/leagues/:id/sport/:id/teams```, how does one specifically get the params for league and sport? In Rails I remember we get the syntactic sugar for params giving us the last id in the URL unless we did something like league.id, and such that lets us specify which params we care about.

Would I need to do ```Sport.find(req.params.id).then(function(sport) {``` and change ```req.params.id``` to ```req.params.sport.id``` and also add a ```League.find(req.params.league.id)```?